WebUI/src/views/Answer.vue

```diff
@@ -461,6 +461,10 @@ async function generate(chatContext: ChatItem[]) {
   if (processing.value || chatContext.length == 0) { return; }
 
   try {
+    const inferenceBackendService: BackendServiceName = textInference.backend === 'IPEX-LLM' ? "ai-backend" : "llamacpp-backend"
```

Comment:
Does this mapping work? Do IPEX-LLM and llamacpp-backend represent the same backend?

Reply:
this was taken from the prior versions. I suppose ipex llm is the text model inference "engine". I however did not fully check